### PR TITLE
docs: improve Introduction docs

### PR DESCRIPTION
**Pull request by jferrettiboke**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -6,17 +6,30 @@ nextPage:
...
+By expanding the range of available icons, Lucide provides users with a broader selection for their projects. Furthermore, as new applications with specific features emerge, Lucide aims to offer a comprehensive icon set to meet diverse project requirements.
 
-Beside design, code is also important. Assets like icons in, for example, web projects can increase the bandwidth usage significantly. With the growing internet, lucide has the responsibility to keep their assets as small as possible. To achieve this, lucide uses SVG compression and specific code architecture for tree-shaking abilities. After tree-shaking, you will only ship the icons you used, helps you to keep the software distribution size to a minimum.
+When designing new icons, the community adheres to a set of design rules to ensure consistency and usability. While creativity is encouraged, maintaining recognizable design conventions is paramount.
```
Comment (karsa-mistmere) [inline docs/index.md:16]:
```suggestion
When designing new icons, the community adheres to a strict set of design rules to ensure consistency and legibility on all sizes. While creativity is encouraged, easily recognizable design is of greater importance to maintain our [design conventions](https://lucide.dev/docs/icon-design-guide).
```

File: docs/index.md [hunk 2]
```diff
@@ -6,17 +6,30 @@ nextPage:
...
 
-With more icons, we simply have more icons to work with in our projects. Also with rising of new applications with specific features, lucide has the goal to provide the complete set for your project.
+Lucide encompasses icons with various variants and states, allowing designers and developers to select the most suitable icon for their needs. If a desired icon is not available, users are encouraged to submit a design request. The Lucide community contributors are dedicated to creating new icons and fulfilling such requests.
 
-When designing new icons, the community is working with a set of design rules. This is to maintain some standards for the icons: recognizable, consistency in style, and readable on all sizes. The community loves creativity in new icons but recognizable design conventions are important.
+By expanding the range of available icons, Lucide provides users with a broader selection for their projects. Furthermore, as new applications with specific features emerge, Lucide aims to offer a comprehensive icon set to meet diverse project requirements.
```
Comment (karsa-mistmere) [inline docs/index.md:14]:
```suggestion
By expanding the range of available icons, Lucide provides users with a broader selection for their projects. Furthermore, as new applications with specific features emerge, Lucide aims to be a comprehensive icon set to meet diverse project requirements.
```

File: docs/index.md [hunk 3]
```diff
@@ -6,17 +6,30 @@ nextPage:
 ---
 
 # Introduction
-Lucide is an open source icon library for displaying icons and symbols in digital and non-digital projects. It consists of 850+ Vector (svg) files. To use these icons, lucide provides several official packages to make it easier to use these icons in projects.
 
-Lucide contains icons with different variants and states. With that, designers and developers can choose the right icon for themselves. If a desired icon doesn't exist yet, you're free to open a design request. The Lucide community contributors will help to provide new icons.
+Lucide is an open-source icon library for displaying icons and symbols in both digital and non-digital projects. It offers a collection of 1000+ vector (SVG) files. To simplify the integration of these icons into projects, Lucide provides several official packages.
```
Comment (karsa-mistmere) [inline docs/index.md:10]:
```suggestion
Lucide is an open-source icon library to be used in both digital and non-digital projects. It offers a collection of 1000+ icons and symbols in the form of vector images (SVG files). To simplify the integration of these icons into projects, Lucide provides several official packages.
```

**Comment by jferrettiboke** [comment]
@karsa-mistmere I totally agree with you! I'll keep this in mind for future docs improvements.

**Comment by ericfennis** [comment]
In the new site #1275 I also updated this text. I tried to split it up a bit to create some sections in text that improve the overview of the page.

Live version: https://lucide-docs.vercel.app/guide/

Maybe we can combine these changes?

This is the text:

# What is Lucide?

Lucide is an open-source icon library that provides 1000+ vector (svg) files for displaying icons and symbols in digital and non-digital projects. The library aims to make it easier for designers and developers to incorporate icons into their projects by providing several official [packages](/packages) to make it easier to use these icons in your project.

## Available Icons

Lucide contains icons with different variants and states, allowing users to choose the most suitable icon for their needs. And if a desired icon isn't available yet, users can open a design request, and the Lucide community contributors will help provide new icons. With more icons to choose from, users have more options to work with in their projects.
Complete Set of Icons

As new applications with specific features arise, Lucide aims to provide a complete set of icons for every project. The community follows a set of design rules when designing new icons. These rules maintain standards for the icons, such as recognizability, consistency in style, and readability at all sizes. While creativity is valued in new icons, recognizable design conventions are important to ensure that the icons are easily identifiable by users.

## Code Optimization

In addition to design, code is also important. Assets like icons can significantly increase bandwidth usage in web projects. With the growing internet, Lucide has a responsibility to keep their assets as small as possible. To achieve this, Lucide uses SVG compression and specific code architecture for tree-shaking abilities. After tree-shaking, you only ship the icons you used, which helps to keep software distribution size to a minimum.

## Official Packages

Lucide's official packages are designed to work on different platforms, making it easier for users to integrate icons into their projects. The packages are available for various technologies, including [Web (Vanilla)](https://lucide.dev/guide/packages/lucide), [React](https://lucide.dev/guide/packages/lucide-react), [React Native](https://lucide.dev/guide/packages/lucide-react-native), [Vue](https://lucide.dev/guide/packages/lucide-vue), [Vue 3](https://lucide.dev/guide/packages/lucide-vue-next), [Svelte](https://lucide.dev/guide/packages/lucide-svelte),[Preact](https://lucide.dev/guide/packages/lucide-preact), [Solid](https://lucide.dev/guide/packages/lucide-solid), [Angular](https://lucide.dev/guide/packages/lucide-angular), [NodeJS](https://lucide.dev/guide/packages/lucide-static#nodejs) and [Flutter](https://lucide.dev/guide/packages/lucide-flutter).

## Community
If you have any questions about Lucide, feel free to reach out to the community. You can find them on [GitHub](https://github.com/lucide-icons/lucide) and [Discord](https://discord.gg/EH6nSts).

**Review by ericfennis — CHANGES_REQUESTED** [review]
@jferrettiboke Can you rebase the PR and look at my last comment?

**Pull request closed without merging**